alerts/openshift-virtualization-operator/WaspNamespaceDetected.md

```diff
@@ -0,0 +1,45 @@
+# WaspNamespaceDetected
+
+## Meaning
+
+This alert fires when there is a duplicate wasp-agent deployment on the cluster
+
+## Impact
+Inconsistent behavior of workload swap configuration.
+
+## Diagnosis
+Identify the duplicate wasp-agent namespace:
+   ```bash
+   oc get ns wasp
+   ```
+
+## Mitigation
+
+1. Delete the wasp-agent Daemonset
+   ```bash
+   oc delete ds wasp-agent -n wasp
+   ```
+2. Delete the wasp-agent service account
+   ```bash
+   oc delete sa wasp -n wasp
+   ```
+3. Delete the wasp-agent cluster role binding
+   ```bash
+   oc delete clusterrolebinding wasp
+   ```
+4. Remove the `wasp` service account from privileged SCC
+   ```bash
+   oc adm policy remove-scc-from-user -n wasp privileged -z wasp
+   ```
```

Comment:
The new wasp-agent will work as expected after this deletion flow or there is also a need to delete its pod so it will be redeployed?